data_submission/src/encipher/mod.rs

```diff
@@ -0,0 +1,82 @@
+use crate::encipher::types::{DecryptRequest, EncryptRequest};
+use reqwest::Client;
+use types::{DecryptResponse, EncryptResponse};
+
+pub mod avail_client;
+pub mod types;
+
+/// Encipher encryption service
+///
+/// # Arguments
+/// * `service_url` - The URL of the Encipher service
+/// * `service_version` - The version of the Encipher service
+pub struct EncipherEncryptionService {
```

Comment:
please rename the folder as `enigma` and not encipher

Reply:
also dont add this entire service as a part of this data submission, just create a root level crate for enigma service and add it to the workspace just like avail crate is written